module/scene-config.js

```diff
@@ -0,0 +1,26 @@
+Hooks.on("renderSceneConfig", (sheet, html, data) => {
```

Comment:
Is this a prefer / good way of adding something into scene config? Looks little bit hacky 🤔
(looking at it on my phone so have to investigate further in Foundry API)

Reply:
It's the preferred way to my knowledge. It's hacky, but this way it works even if modules try to further change the scene config in some way, without running into conflicts with multiple templates overriding each other for the same config. Or a module could add an entirely separate scene config template, and this would *still* work as long as it had a journal slot for my code to find.

Reply:
I see. Make sense, will try to look into it in case there is some "register scene section" API or something. But if not this is probably truly best we can do. Thanks for explanation 👍